packages/babel-parser/src/plugins/placeholders.js

```diff
@@ -251,6 +251,21 @@ export default (superClass: Class<Parser>): Class<Parser> =>
       return super.parseExport(node);
     }
 
+    isExportDefaultSpecifier(): boolean {
+      if (this.match(tt._default)) {
+        const next = this.nextTokenStart();
+        if (this.isUnparsedContextual(next, "from")) {
+          if (
+            this.input.substr(this.nextTokenStartSince(next + 4), 2) ===
+            tt.placeholder.label
```

Comment:
```suggestion
            
            this.input.startsWith(tt.placeholder.label, this.nextTokenStartSince(next + 4))
```

Since you are not hard-coding `%%` but using `tt.placeholder.label`, let's also not hard-code it's length (`2`)!